docs/DeveloperGuide.md

```diff
@@ -406,6 +411,59 @@ Implementing the FileChooser library allows us to create a File Explorer GUI sim
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
+---
+### Suggest Show
+
+#### What it does
+
+Returns a single show from the current displayed list of shows
+
+#### Implementation
+
+After executing the suggest command, it would create a SuggestCommand object is created.
+Then, LogicManager will then execute the given SuggestCommand. Upon execution of the SuggestCommand's execute method,
+it will obtain the currently displayed list of show via the model's `getFilteredShowList()`method.
+from the `model` object.
+
+In the event that the list is empty, this would cause an error message. Informing users that
+there are no shows currently in the displayed show list. Similarly, if there is only 1 show 
+present in the list, it would inform the user that there is only 1 show in the current displayed 
```

Comment:
In line 428, you had mentioned users. However, in line 430, you mentioned the user. There is some inconsistency here. Should the focus be the user (singular) or users (plural)?